pyproject.toml

```
@@ -0,0 +1,116 @@
+# build-system
+# - ref: https://peps.python.org/pep-0517/
+#
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+# project
+# - ref 1: https://peps.python.org/pep-0621/
+# - ref 2: https://hatch.pypa.io/latest/config/metadata/#project-metadata
+#
+[project]
+name = "jupyterhub-pytest-plugin"
+description = "A reusable JupyterHub pytest plugin"
+readme = "README.md"
+requires-python = ">=3.5"
```

Let's set a min version of 3.8 since we're starting fresh
```suggestion
requires-python = ">=3.8"
```